src/crimson/os/alienstore/CMakeLists.txt

```diff
@@ -76,5 +76,5 @@ target_link_libraries(crimson-alienstore
     crimson-alien-common
     ${BLKID_LIBRARIES}
     ${UDEV_LIBRARIES}
-    crimson
+    seastar
```

Comment:
OK, this seems to answer my question – there is nothing that would require having the 2 flags; the `crimson` things aren't part of alien anymore.